VTUL/VIVO_VT_Theme
# Fixes read more bug

**Resolves bug that prevented individual overview section from being expandable.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1772) (:star:)

# What does this Pull Request do? (:star:)
Resolves bug that prevented individual overview section from being expandable.

# What's the changes? (:star:)
* loads `moreLessController.js` script
* calls `truncate()` on `overview-value` div instead of using `substring()`

# How should this be tested?
* Add a faculty member
* Add text to the faculty member's overview section that is greater than 500 characters
* Check that the "read more/read less" functionality works.

# Additional Notes:
* branch: `LIBTD-1772`

# Interested parties
@yinlinchen 

(:star:) Required fields
